VisualPinball.Unity/VisualPinball.Unity/VPT/ISwitchableAuthoring.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Visual Pinball Engine
+// Copyright (C) 2020 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+namespace VisualPinball.Unity
+{
+	public interface ISwitchableAuthoring : IIdentifiableItemAuthoring
+	{
+	}

```

Comment:
This will contain the switch events, right?

Reply:
I believe so. Was hoping to get all the groundwork in and then we could start on that.